app/views/shared/error_page.html.erb

```diff
@@ -0,0 +1,6 @@
+<% content_for(:html_title) { code + ' ' + HttpStatusHelper::ERROR_CODE[code]}%>

```

Comment:
I thought about it and I think it's better to remove the code and only display the name (e.g., "Forbidden" instead of "403 Forbidden"), since the code won't be meaningful for regular users. Same with the `h1` text below.